### ISSUE TITLE
Zombies not moving on Windows

### ISSUE THREAD

**Issue by egonelbre** [loov/zombieroom#1]
Comment from https://www.reddit.com/r/gogamedev/comments/5hv9uy/loovzombiesonice/deccof2/

The zombies' positions don't change unless I move the window. And once it is moved they don't automatically update. I can move my character around and kill the zombies, but they stand in place.

Happens with the 32bit and 64bit windows ludum dare versions.

**Comment by phanirithvij** [comment]
@egonelbre this issue is not in the binaries downloaded from v0.1 https://github.com/adinfinit/zombies-on-ice/releases/tag/v0.1

**Comment by egonelbre** [comment]
Yeah, this might be something related to the particular machine. It worked for me at that time...

However, since it's pretty much unactionable at this moment, I'll close it.

**Comment by phanirithvij** [comment]
@egonelbre I can reproduce with the latest master, but it is working fine with the older v0.1

**Comment by egonelbre** [comment]
Can you try find out where it gets stuck and whether the main loop is still running? I'm unfortunately unable to repro the issue to debug.

**Comment by egonelbre** [comment]
Alternatively, a specific commit might help as well.

**Comment by phanirithvij** [comment]
It is with the glfw version try my fork's master https://github.com/phanirithvij/zombies-on-ice on windows (v3.1 glfw, v.3.3 gl), without a joystick it works normally. But when the game starts with a joystick connected **or** if a joystick is connected in the middle of the game the zombies stop moving until the window is resized and will move extremely slowly afterwords.

With the current master in this main repo, the same thing happens even without the joystick but with the plain keyboard.

https://github.com/adinfinit/zombies-on-ice/commit/6ee12bf1a994ce988644933aa5ca80887523a7c6 is where it started

I Will get back to you in a few hours.

**Comment by egonelbre** [comment]
Maybe it needs https://pkg.go.dev/github.com/go-gl/glfw/v3.3/glfw#SwapInterval?

**Comment by egonelbre** [comment]
The only thing close enough I saw was: https://github.com/glfw/glfw/commit/8309e0ecb09fe5cf670dff5df1e7ca71821c27bd

So maybe DWM or VSync issue?

**Comment by phanirithvij** [comment]
I don't really much glfw but the more zombies there are the more each of their speed is.
